lib/federation/docker-compose.yml

```diff
@@ -32,3 +36,8 @@ services:
         SERVICE_NAME: ${SERVICE_NAME}
     extra_hosts:
         - "${CANDIG_DOMAIN}:${LOCAL_IP_ADDR}"
+    healthcheck:
+      test: ["CMD", "python", "/app/federation/candig_federation/healthcheck.py"]
```

Comment:
I think the same for this healthcheck.py, you will need to commit it in federation and make a PR in that repo before updating in CanDIGv2

Reply:
Added! Here is the [link](https://github.com/CanDIG/federation_service/pull/48)!